sklearn/ensemble/_forest.py

```diff
@@ -355,7 +355,7 @@ def fit(self, X, y, sample_weight=None):
                              'len(estimators_)=%d when warm_start==True'
                              % (self.n_estimators, len(self.estimators_)))
 
-        elif n_more_estimators == 0:
+        if n_more_estimators == 0:
```

Comment:
I would still keep an elif here

Reply:
I would also keep it an `elif` because we have an `else` just after

Reply:
done